Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml

```diff
@@ -0,0 +1,1268 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "20":
+    id: "20"
+    taskid: 5d371f29-3a4c-43c5-8f71-b383db2e5320
+    type: title
+    task:
+      id: 5d371f29-3a4c-43c5-8f71-b383db2e5320
+      version: -1
+      name: CrowdStrike Falcon
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "10"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 2460,
+          "y": 575
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "22":
+    id: "22"
+    taskid: f7f190b9-5a39-4d8a-83a5-77d5a023f0d4
+    type: condition
+    task:
+      id: f7f190b9-5a39-4d8a-83a5-77d5a023f0d4
+      version: -1
+      name: Is Active Directory Query v2 enabled?
+      description: Checks if there is an active instance of the Active Directory Query v2 integration enabled.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "23"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Active Directory Query v2
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: active
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": 710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "23":
+    id: "23"
+    taskid: 23c75906-3b4e-42e8-8e32-c883d5f06fb3
+    type: regular
+    task:
+      id: 23c75906-3b4e-42e8-8e32-c883d5f06fb3
+      version: -1
+      name: Get host information from Active Directory
+      description: |-
+        Uses Active Directory to retrieve detailed information about the computer of the account specified. The computer can be specified by name, email address, or as an Active Directory Distinguished Name (DN).
+        If no filters are specified, returns all computers.
+      script: Active Directory Query v2|||ad-get-computer
```

Comment:
Use 'ad-get-computer' instead of 'ad-get-computer (Active Directory Query v2)'

Reply:
Done.